src/server/app/server.ts

```diff
@@ -34,6 +34,11 @@ app.get('/data', function (req, res) {
     });
 });
 
+app.post('/boards/:id', function (req, res) {
```

Comment:
This is just the stub function to test the client with and should not be merged onto master. The client will need to run with the implementation that the server team has made. :)